BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -137,16 +137,26 @@
     <string name="format_options">Format Options</string>
     <string name="format_arrow">Format Arrow</string>
 
-
     <!-- Export Dialog Alert Window Descriptions -->
     <string name="storage_permission_needed">Zugriff erforderlich</string>
     <string name="storage_permission_alert_msg">
         Zugriff auf den Gerätespeicher erforderlich,
         um Dateien aus dieser App zu exportieren.
+        Erlaube den Zugriff und versuche nochmal die
+        Dateien zu exportieren.
     </string>
-    <string name="storage_permission_ok_btn">OK</string>
+    <string name="ok">OK</string>
     <string name="storage_permission_cancel_btn">Abbrechen</string>
     <string name="storage_permission_denied">Zugriff verweigert</string>
+    <string name="file_write_failed">File write failed:</string>
+    <string name="exported_file_stored_in">Exportierte Datei ist gespeichert unter:</string>
+
+    <!-- Export: Empty Booklist -->
+    <string name="empty_booklist">Die Bücherliste ist leer</string>
+    <string name="empty_booklist_description">
+        Bitte füge ein/ mehrere Bücher in die
```

Comment:
Missing comma before "um", please use "Regal" instead of "Liste" and before and after a "/", no spaces should be set, see Duden:

"Unsere Empfehlung: Steht nur jeweils ein Wort vor und nach dem Schrägstrich, setzen Sie kein Leerzeichen, z. B. April/Mai."
https://www.duden.de/sprachwissen/sprachratgeber/Der-Schr%C3%A4gstrich 


Suggested wording: "Bitte stelle Bücher in das Regal, um den Export zu ermöglichen."